client/lib/products-values/constants.js

```diff
@@ -69,6 +69,24 @@ export const JETPACK_PRODUCT_DISPLAY_NAMES = {
 	...JETPACK_BACKUP_PRODUCT_DISPLAY_NAMES,
 };
 
+export const JETPACK_BACKUP_PRODUCT_DAILY_TAG_LINE = translate(
```

Comment:
Isn't `tagline` a single word? If so, let's ditch that underscore in all those constants.

Reply:
The dictionary states that the primary form is a "tag line" but a "tagline" as a single word is also accepted. Since it's simpler, I'm going to change it here and in other locations affected by this PR to a single word form 🙂

Reply:
Fair enough that they're both correct. I guess I mean that the single word form is more common around here in the WP context 😉